.storybook/main.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
   "stories": [
-    "../src/**/*.stories.mdx",
-    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+    "../packages/**/*.stories.mdx",
+    "../packages/**/*.stories.@(js|jsx|ts|tsx)"
```

Comment:
Once we are ready with TS (#7), this should be only `ts|tsx`, right?

Reply:
Yes.